rpm/packagescanner.go

```diff
@@ -240,6 +240,7 @@ const queryFmt = `%{name}\n` +
 	`%{payloaddigestalgo}:%{payloaddigest}\n` +
 	`%{sigpgp:pgpsig}\n` +
 	`%{sourcerpm}\n` +
+	`%{RPMTAG_MODULARITYLABEL}\n` +
```

Comment:
Do we need to inform users to use a specific rpm version (or geater) to support this module query?

Reply:
Yes the label was introduced in rpm-4.15.0-alpha.

Reply:
noted. thanks.